classes/class-wc-connect-note-dhl-live-rates-available.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * WooCommerce Shipping note: DHL live rates available.
+ *
+ * Only for legacy customers that had the feature available.
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+if ( ! class_exists( 'WC_Connect_Note_Dhl_Live_Rates_Available' ) ) {
+
+	/**
+	 * Note to inform WooCommerce Shipping users with legacy live rates about DHL live rates.
+	 */
+	class WC_Connect_Note_DHL_Live_Rates_Available {
+
+		use Automattic\WooCommerce\Admin\Notes\NoteTraits;
```

Comment:
It looks like this got introduced (with the `get_note` method) in this PR https://github.com/woocommerce/woocommerce-admin/pull/4340 in May 2020
So anybody testing with WC~< 4.2 would get a fatal error. We require at least WC 3.0 - should we add some checks for this? Or should we update our headers?

Reply:
That's a good point, bumping it only for a notice it is not really worth it.
I will add a check.

Reply:
Added a check for the trait.